Resources/public/js/models/ez-locationmodel.js

```diff
@@ -303,6 +303,15 @@ YUI.add('ez-locationmodel', function (Y) {
         },
 
         /**
+         * @method _ïsRootLocation
```

Comment:
there is a typo in the method name. The comments explaining what it does is missing.
Also since you moved it in the model (which is not what I meant, but it seems like a good idea) it needs to be public and unit tested in the location model tests.